test/fixtures/pkg/kubectl/cmd/set/multi-resource-yaml.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: v1
```

Comment:
I think we could re-use some testing yaml files, instead of creating new ones.

`test/e2e/testing-manifests/statefulset/mysql-upgrade/service.yaml` can be used for your testing.

Reply:
this fixtures dir will distinguish diff command, i add it to `set command` dir

Reply:
not find suitable yaml can use ~.~ this yaml must have two resources and have `template` field.